script.js

```diff
@@ -0,0 +1,89 @@
+let playerScore = 0;
+let computerScore = 0;
+let playerChoice;
+
+function getComputerChoice() {
+  let computerChoice = Math.floor(Math.random() * 3);
+  if (computerChoice === 0) {
+    computerChoice = "rock";
+  } else if (computerChoice === 1) {
+    computerChoice = "paper";
+  } else {
+    computerChoice = "scissors";
+  }
+  return computerChoice;
+}
+
+const buttons = document.querySelectorAll(".buttons button");
+buttons.forEach((buttons) => buttons.addEventListener("click", processClick));
+
+function processClick() {
+  playerChoice = this.id;
+  playRound();
+}
+
+const reset = document.querySelector(".reset-button");
+reset.addEventListener("click", resetGame);
+
+function resetGame() {
+  playerScore = 0;
+  computerScore = 0;
+  playerChoice = null;
+  document.getElementById("c-score").innerHTML = computerScore;
```

Comment:
I recommend writing a function like displayScores that updates c-score and p-score and call that function here and on line 71. (It's not necessary to, but in general code that is reused is a great place to think about using another function so that the code doesn't need to be repeated.  One example of an improvement is if you decided to change the ids of the elements from c-score and p-score to something else you'd have to change that code in two places now)